project/main.py

```diff
@@ -17,14 +17,16 @@ def update_access_key(key):
 def set_DryRun(bool):
     values.DryRun = bool
 
-def readConfigFile(path):
+def readConfigFile(path,logging):
```

Comment:
Probably don't need to pass in logging

Reply:
From looking at the code it doesn't look like the logging object is global so it may not work in these functions without passing it or making it global?  Let me know if I'm wrong

Reply:
This function is only called from main, and logging is imported into main. This should be fine.

Reply:
Yeah but you added some logging to the readConfigFile function which won't work without a logging handler

Reply:
It is working in testing after no longer passing logging to readConfigFile.

Reply:
You ran it with debug logging turned on? If so, then it should be getting the object okay.